v2.1/admin-ui-hardware-dashboard.md

```diff
@@ -8,29 +8,13 @@ The **Hardware** dashboard lets you monitor CPU usage, disk throughput, network
 
 The **Hardware** dashboard displays the following time series graphs:
 
-## User CPU Percent
+## CPU Percent
 
-<img src="{{ 'images/v2.1/admin_ui_user_cpu.png' | relative_url }}" alt="CockroachDB Admin UI User CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v2.1/admin_ui_cpu_percent.png' | relative_url }}" alt="CockroachDB Admin UI CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
 
-- In the node view, the graph shows the amount of CPU resources used by the CockroachDB process for the selected node.
+- In the node view, the graph shows the amount of CPU resources used by the CockroachDB process as well as the CockroachDB system-level operations for the selected node.
```

Comment:
"for the selected node" seems a bit weird, since we are showing all nodes; there isn't really a way to "select" one. How about "for each node"? (This comment applies to your docs for every graph)

Reply:
Oh, I guess you mean selecting "cluster" vs selecting one node in the selector on the top left. I guess that makes sense, although it seems a little heavyweight to repeat both forms for every single graph, as opposed to a little blurb at the top about how this works.

(Then again, "how this works" will change when we land a toggle)